exist-core/src/test/xquery/dates/convert-dates.xqm

```diff
@@ -25,68 +25,68 @@ module namespace fd="http://exist-db.org/xquery/test/convert-dates";
 declare
     %test:args("2012-06-26T23:14:22.566+02:00")
     %test:assertEquals("11:14 pm on Tuesday, June 26th, 2012")
-function fd:format-dateTimeStamp($date as xs:dateTimeStamp) {
+function fd:format-dateTime($date as xs:dateTime) {
     format-dateTime($date, "[h00]:[m00] [P] on [FNn], [MNn] [D1o], [Y]", "en", (), ())
 };
 
-(: verify that we can't convert xs:string without timezone to :xs:dateTimeStamp  :)
+(: verify that we can't convert xs:string without timezone to :xs:dateTime :)
 declare
     %test:args("2012-06-26T23:14:22.566")
     %test:assertError
-function fd:convert-date-time-stamp($date as xs:dateTimeStamp) {
+function fd:convert-date-time($date as xs:dateTime) {
     format-dateTime($date, "[h00]:[m00] [P] on [FNn], [MNn] [D1o], [Y]", "en", (), ())
 };
 
 declare
     %test:args("2022-05-17T16:24:06.003+02:00", "PT2H")
     %test:assertEquals("2022-05-17T18:24:06.003+02:00")
-function fd:add-test($date as xs:dateTimeStamp, $duration as xs:dayTimeDuration) {
+function fd:add-test($date as xs:dateTime, $duration as xs:dayTimeDuration) {
     $duration + $date
 };
 
 
-(: verify that fn:current-dateTime() return type is xs:dateTimeStamp  :)
+(: verify that fn:current-dateTime() return type is xs:dateTime  :)
 declare
     %test:assertEquals("true")
 function fd:current-dateTime-type() {
-    fn:current-dateTime() instance of xs:dateTimeStamp
+    fn:current-dateTime() instance of xs:dateTime
 };
 
 declare
     %test:args("2022-05-17T16:24:06.003+02:00", "PT2H")
     %test:assertEquals("true")
-function fd:return-type-test($date as xs:dateTimeStamp, $duration as xs:dayTimeDuration) {
-    ($duration + $date) instance of xs:dateTimeStamp
+function fd:return-type-test($date as xs:dateTime, $duration as xs:dayTimeDuration) {
+    ($duration + $date) instance of xs:dateTime
 };
 
 declare
-    %test:args("not-a-dateTimeStamp")
+    %test:args("not-a-dateTime")
     %test:assertError
-function fd:not-a-dateTimeStamp($date as xs:dateTimeStamp) {
-    $date instance of xs:dateTimeStamp
+function fd:not-a-dateTime($date as xs:dateTime) {
+    $date instance of xs:dateTime
 };
 
 declare
     %test:args("2022-05-17T17:16:00.000")
     %test:assertError
-function fd:not-a-dateTimeStamp2($date as xs:dateTimeStamp) {
-    $date instance of xs:dateTimeStamp
+function fd:not-a-dateTime2($date as xs:dateTime) {
+    $date instance of xs:dateTime
 };
 
 declare
     %test:args("2022-05-17T17:16:00.000Z")
     %test:assertEquals("true")
-function fd:create-dateTimeStamp($date as xs:dateTimeStamp) {
-    $date instance of xs:dateTimeStamp
+function fd:create-dateTime($date as xs:dateTime) {
+    $date instance of xs:dateTime
 };
 
 declare
     %test:args("2022-05-17T17:16:00.000+01:00")
     %test:assertEquals("true")
-function fd:create-dateTimeStamp2($date as xs:dateTimeStamp) {
-    $date instance of xs:dateTimeStamp
+function fd:create-dateTime2($date as xs:dateTime) {
+    $date instance of xs:dateTime
```

Comment:
The original test was valid, as `xs:dateTimeStamp` is required to have an explicit timezone offset, as noted above.

Reply:
This has also now been undone.